src/main.ts

```diff
@@ -2,23 +2,25 @@ import './config/env'; // Important to be before other dependencies
 import throng from 'throng';
 import getRedisInfo from './config/redis-info';
 import * as PrivateKey from 'probot/lib/private-key';
-import { createProbot } from 'probot';
+import { Probot } from 'probot';
 import App from './configure-robot';
 import bunyan from 'bunyan';
 import { exec } from 'child_process';
 import { initializeSentry } from './config/sentry';
+import { getLogger } from './config/logger'
 
 const isProd = process.env.NODE_ENV === 'production';
 const { redisOptions } = getRedisInfo('probot');
 
-const probot = createProbot({
+
+const probot = new Probot({
```

Comment:
Probably keep this as `createProbot` unless you're using something with the probot constructor options directly.